Storage/src/Connection/RetryTrait.php

```diff
@@ -44,37 +44,37 @@ trait RetryTrait
      * (Idempotent)
      * @var array
      */
-    private $idempotentOps = [
-        'bucket_acl.get' => true,
-        'bucket_acl.list' => true,
-        'buckets.delete' => true,
-        'buckets.get' => true,
-        'buckets.getIamPolicy' => true,
-        'buckets.insert' => true,
-        'buckets.list' => true,
-        'buckets.lockRetentionPolicy' => true,
-        'buckets.testIamPermissions' => true,
-        'default_object_acl.get' => true,
-        'default_object_acl.list' => true,
-        'hmacKey.delete' => true,
-        'hmacKey.get' => true,
-        'hmacKey.list' => true,
-        'notifications.delete' => true,
-        'notifications.get' => true,
-        'notifications.list' => true,
-        'object_acl.get' => true,
-        'object_acl.list' => true,
-        'objects.get' => true,
-        'objects.list' => true,
-        'serviceaccount.get' => true,
+    public static $idempotentOps = [
+        'bucket_acl.get',
+        'bucket_acl.list',
+        'buckets.delete',
+        'buckets.get',
+        'buckets.getIamPolicy',
+        'buckets.insert',
+        'buckets.list',
+        'buckets.lockRetentionPolicy',
+        'buckets.testIamPermissions',
+        'default_object_acl.get',
+        'default_object_acl.list',
+        'hmacKey.delete',
+        'hmacKey.get',
+        'hmacKey.list',
+        'notifications.delete',
+        'notifications.get',
+        'notifications.list',
+        'object_acl.get',
+        'object_acl.list',
+        'objects.get',
+        'objects.list',
+        'serviceaccount.get',
     ];
 
     /**
      * The operations which can be retried with specific conditions
      * (Conditionally idempotent)
      * @var array
      */
-    private $condIdempotentOps = [
+    public static $condIdempotentOps = [
```

Comment:
In PHP 8.2, this results in a deprecated warning:

```php
PHP Deprecated:  Accessing static trait property RetryTrait::$condIdempotentOps is deprecated, it should only be accessed on a class using the trait
```

We should not be exposing static methods on traits, as this goes against PHP best practices for how traits are supposed to be used.

Reply:
@yash30201